lib/datasource/gitlab-packages/readme.md

```diff
@@ -0,0 +1,32 @@
+[Gitlab Generic Packages Repository](https://docs.gitlab.com/ee/user/packages/generic_packages/) supports looking up package versions from all type of packages registry supported by Gitlab and can be used in combination with [regex managers](https://docs.renovatebot.com/modules/manager/regex/) to keep dependencies up-to-date which are not specifically supported by Renovate.
+
+To specify which specific repository should be queried when looking up a package, the `registryUrl` has to be set like this: `https://gitlab.com/user/project`. As an example, `https://gitlab.com/gitlab-org/ci-cd/package-stage/feature-testing/new-packages-list` would look for packages in the generic packages repository of the
+`gitlab-org/ci-cd/package-stage/feature-testing/new-packages-list` project.
+
+**Usage Example**
+
+A real world example for this specific datasource would be maintaining packages versions in a config file. This can be achieved by configuring a generic regex manager in `renovate.json` for files named `versions.ini`:
```

Comment:
```suggestion
A real world example for this specific datasource would be maintaining packages versions in a config file.
This can be achieved by configuring a generic regex manager in `renovate.json` for files named `versions.ini`:
```